contrib/mesos/ci/run.sh

```diff
@@ -42,13 +42,30 @@ else
 fi
 echo "${DOCKER_BIN_PATH}"
 
+# from hack/lib/golang.sh and build/common.sh
+readonly KUBE_GO_PACKAGE=k8s.io/kubernetes
+
+readonly LOCAL_OUTPUT_ROOT="${KUBE_ROOT}/_output"
+readonly LOCAL_OUTPUT_SUBPATH="${LOCAL_OUTPUT_ROOT}/dockerized"
+readonly LOCAL_OUTPUT_BINPATH="${LOCAL_OUTPUT_SUBPATH}/bin"
+
+readonly OUTPUT_BINPATH="${CUSTOM_OUTPUT_BINPATH:-$LOCAL_OUTPUT_BINPATH}"
```

Comment:
probably wise to set a `CUSTOM_OUTPUT_BINPATH` to avoid nested volume mounts -- it's only going to hurt us later